tests/Tests.elm

```diff
@@ -19,27 +19,27 @@ both : (a -> b) -> (a, a) -> (b, b)
 both f (x, y) = (f x, f y)
 
 tests : Test
-tests = 
-  describe "Stack tests" 
-    [ describe "empty" 
-        [ test "has a size of 0" <| 
+tests =
+  describe "Stack tests"
+    [ describe "empty"
+        [ test "has a size of 0" <|
             \() ->
                 Stack.size Stack.empty |> Expect.equal 0
-        
+
```

Comment:
sorry for these changes in diff. I have my editor configured to automatically strip trailing white spaces and enforce unix style file ends (made to not mess with C preprocessor). Let me know if you want me to revert these changes.

Reply:
Not an issue!